apps/server/src/modules/group/repo/group.repo.ts

```diff
@@ -205,4 +163,39 @@ export class GroupRepo extends BaseDomainObjectRepo<Group, GroupEntity> {
 
 		return domainObjects;
 	}
+
+	private buildPipeline(skip?: number, limit?: number, nameQuery?: string): unknown[] {
+		let nameQueryMatch = {};
+		const pipeline: unknown[] = [];
+
+		if (nameQuery && StringValidator.isNotEmptyString(nameQuery, true)) {
+			const escapedName = nameQuery.replace(MongoPatterns.REGEX_MONGO_LANGUAGE_PATTERN_WHITELIST, '').trim();
+			if (StringValidator.isNotEmptyString(escapedName, true)) {
```

Comment:
What happens if the replacement fails? 
Shouldnt we introduce a try-catch-block here?

Reply:
After consideration, there should be no way a String.replace would fail on a string.